src/panels/lovelace/cards/hui-markdown-card.ts

```diff
@@ -58,19 +58,17 @@ export class HuiMarkdownCard extends LitElement implements LovelaceCard {
 
     this._config = config;
     this._disconnect().then(() => {
-      if (this._hass) {
-        this._connect();
-      }
+      this._connect();
     });
   }
 
-  public disconnectedCallback() {
-    this._disconnect();
+  public connectedCallback() {
+    super.connectedCallback();
+    this._connect();
```

Comment:
Do we always have `hass` and config when connected?

Reply:
We check in the connect function

Reply:
I know, but if we don't have `hass` then, we will never connect.

Reply:
So you are saying add it to first updated as well?